torch/quantization/fake_quantize.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+from torch.nn import Module
+from .._jit_internal import weak_module
+from .observer import Observer, observer
+
+@weak_module
+class FakeQuantize(Module):
+    ''' Simulate the quantize and dequantize operations in training time.
+    Args:
+        `qconfig`: object that encodes configuration info for quantization
+        `observer_module`: Observer module that records stats of weights and
+        activations
+        `calcqparam`: A function that calculates quantization parameters
+        given the stats
+    '''
+
+    def __init__(self, qconfig, quant_delay=0):
+        super(FakeQuantize, self).__init__()
+        assert qconfig['qscheme'] == torch.per_tensor_affine
+        observer_factory = observer(Observer, {
```

Comment:
why the args are passed as dict?

Reply:
we should probably not call this qconfig, it has some extra arguments like `quant_min` and `quant_max`